lib/bolt/cli.rb

```diff
@@ -100,6 +109,10 @@ def parse_command
       # This part aims to handle both `bolt <mode> --help` and `bolt help <mode>`.
       remaining = handle_parser_errors { parser.permute(@argv) } unless @argv.empty?
       if @argv.empty? || help?(remaining)
+        # If the subcommand is not enabled, display the default
+        # help text
+        options[:subcommand] = nil unless commands.include?(options[:subcommand])
```

Comment:
I think it'd be a better experience if we instructed users on how to enable the module subcommand and linked to docs, rather than just acting like it doesn't exist. If I had heard of the module subcommand (through slack or the docs or something) and went to try it, I wouldn't know where to go to learn more about why it wasn't showing up at all.  We can include caveats about it being experimental both in the message and the docs.

Reply:
This is only preventing users from seeing the help text for a feature-flagged command. It won't prevent users from seeing the help text for the `module` command if `modules` is not configured.

As for updating the help text itself to include links to the docs, that can happen once the docs are finished and we know what the URL to the page is.